src/main/java/uk/gov/hmcts/reform/sendletter/SendLetterConfiguration.java

```diff
@@ -1,21 +1,18 @@
 package uk.gov.hmcts.reform.sendletter;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import feign.codec.Decoder;
 import feign.jackson.JacksonDecoder;
-import org.springframework.cloud.netflix.feign.EnableFeignClients;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
-import org.springframework.context.annotation.Scope;
 
 @Configuration
-@EnableFeignClients(basePackages = "uk.gov.hmcts.reform.sendletter")
```

Comment:
why isn't this needed?

Reply:
one of configuration is already doing it for the same package.

Reply:
```@Configuration
@ConditionalOnProperty(prefix = "send-letter", name = "api.url")
@EnableFeignClients(basePackages = "uk.gov.hmcts.reform.sendletter")
public class SendLetterAutoConfiguration {```